iocage/lib/Jail.py

```diff
@@ -1238,27 +1244,27 @@ def env(self):
         return jail_env
 
     @property
-    def identifier(self):
+    def identifier(self) -> str:
```

Comment:
IIRC, we discussed sometime, somewhere, that this and `name` need consistent (but different) validation.